[15.0] [MIG] partner_time_to_pay + improvements (#249)
OCA/account-invoice-reporting: partner_time_to_pay/readme/USAGE.rst

> * Go to the Accounting tab of a partner

Reviewer: ```suggestion
* Go to the *Sales & Purchase* tab of a partner
```

> * There is 4 new fields. These fields represent the average days to pay for

Reviewer: ```suggestion
* There are 4 new fields. These fields represent the average days to pay for
```

  this year (YTD) and since the first invoice (lifetime).

> * Adds a new field to invoices to check the Full Reconcile Payment Date

Reviewer: ```suggestion
* Adds a new field in the invoices form to check the Full Reconcile Payment Date
```
